src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -125,9 +126,8 @@ func (f *flushState) getLastFlushAt() time.Time {
 }
 
 type writerState struct {
-	writer         commitLogWriter
-	writerExpireAt time.Time
-	activeFile     *File
+	writer     commitLogWriter
+	activeFile *persist.CommitlogFile
```

Comment:
Yeah, can we make this `persist.CommitLogFile` for consistency?  i.e. there is a `CommitLogWriter` etc so there is already precedence for camelcasing the two words CommitLog